core/dbt/cli/requires.py

```diff
@@ -41,14 +41,15 @@ def wrapper(*args, **kwargs):
         ctx.obj["flags"] = flags
         set_flags(flags)
 
-        # Tracking
         initialize_from_flags(flags.SEND_ANONYMOUS_USAGE_STATS, flags.PROFILES_DIR)
```

Comment:
does this (sets up active_user) need to happen logger initialization, or just before tracking? If it's only necessary for tracking, let's move it down where tracking is initialized for readability

Reply:
Sounds good.